### PR TITLE
Add parser for CESM history files

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

File: ecgtools/parsers/cesm.py [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
+    'pop.h.nday1': {'component': 'ocn', 'frequency': 'day_1'},
+    'pop.h.nyear1': {'component': 'ocn', 'frequency': 'year_1'},
+    'pop.h.ecosys': {'component': 'ocn', 'frequency': 'month_1'},
+    'pop.h.ecosys.nyear1': {'component': 'ocn', 'frequency': 'year_1'},
+    'cice.h': {'component': 'ice', 'frequency': 'month_1'},
+    'cice.h1': {'component': 'ice', 'frequency': 'day_1'},
+    'cism.h': {'component': 'glc', 'frequency': 'year_1'},
+    'cism.h1': {'component': 'glc', 'frequency': 'month_1'},
+    'ww3.h': {'component': 'wav', 'frequency': 'month_1'},
+}
```
Comment (andersy005) [inline ecgtools/parsers/cesm.py:42]: @mgrover1, this is the list you gave me earlier today. Is it true that it's missing some streams? If so, could you point me to the documentation page with all the streams? Thanks!

Comment (mgrover1) [reply ecgtools/parsers/cesm.py:42]: Most of them are in there - the only ones that are missing are some the additional 'rof' streams in https://github.com/NCAR/CESM_postprocessing/blob/master/Config/config_timeseries.xml

Comment (mgrover1) [reply ecgtools/parsers/cesm.py:42]: It looks good to start with though!

Comment (andersy005) [reply ecgtools/parsers/cesm.py:42]: Sounds good... I will use the existing list (the one you gave me) for the time being

**Pull request merged**